src/pybind/mgr/dashboard/frontend/src/app/ceph/block/block.module.ts

```diff
@@ -149,8 +149,19 @@ const routes: Routes = [
   {
     path: 'mirroring',
     component: RbdMirroringComponent,
-    canActivate: [FeatureTogglesGuardService],
-    data: { breadcrumbs: 'Mirroring' },
+    canActivate: [FeatureTogglesGuardService, ModuleStatusGuardService],
+    data: {
+      moduleStatusGuardConfig: {
+        uiApiPath: 'block/mirroring',
+        redirectTo: 'error',
+        header: 'RBD mirroring is not configured',
+        button_name: 'Configure RBD Mirroring',
+        button_title: 'This will create rbd-mirror service and a replicated RBD pool',
```

Comment:
BTW, we need the `localize` here for i18n.

Reply:
Done, thanks